src/classes/BDI_DataImportDeleteBTN_CTRL.cls

```diff
@@ -42,6 +42,18 @@ public with sharing class BDI_DataImportDeleteBTN_CTRL {
     /** @description Action type specified to delete all Data Import records */
     public static final String ACTION_DELETE_ALL = 'deleteAll';
 
+    @TestVisible
```

Comment:
One last comment - it looks like with the shift in approach we no longer need these vars to exist, am I reading that right?

Reply:
fixed!